### DISCUSSION TITLE
Is spoo.me really free forever?

### DISCUSSION THREAD

**Discussion by xarmaan** [spoo-me/spoo#1] [Q&A] [answered]
Can I use this in production mode?
Earlier I was using Firebase Dynamic Links which got shutdown.
Can spoo.me really provide everything for free as said?

**Comment by Zingzy** [comment, marked as answer]
> Can I use this in production mode? Earlier I was using Firebase Dynamic Links which got shutdown. Can spoo.me really provide everything for free as said?

Absolutely! You're good to use spoo.me in production – it's currently free. We might explore premium features down the road, but the core service will stay free. If there are any changes, we'll keep you in the loop. Go ahead and make the most of it!
